app/views/default/_proposal_properties.html.erb

```diff
@@ -10,7 +10,11 @@
         <strong><%= name %>
       </td>
       <td>
-        <%= property_to_s(value) %>
+        <% if name == "Link to product" %>
+          <%= link_to truncate_link(value), value %>
+        <% else %>
+          <%= property_to_s(value) %>
+        <% end %>
```

Comment:
Wondering if there is another way to go about this...I don't think we want to hard-code that one property name in this partial, but I could see the use for truncating certain fields.